src/EditorFeatures/CSharpTest/Classification/SyntacticClassifierTests_Preprocessor.cs

```diff
@@ -1119,6 +1119,24 @@ public async Task ShortDiscardInAssignment()
                 expected: Classifications(Identifier("_"), Operators.Equals, Number("1"), Punctuation.Semicolon));
         }
 
+        [Fact]
+        public async Task UnderscoreInLambda()
+        {
+            await TestInMethodAsync(
+                code: @"x = (_) => 1;",
+                expected: Classifications(Identifier("x"), Operators.Equals, Punctuation.OpenParen, Parameter("_"), Punctuation.CloseParen,
+                    Operators.EqualsGreaterThan, Number("1"), Punctuation.Semicolon));
+        }
+
+        [Fact]
+        public async Task DiscardInLambda()
+        {
+            await TestInMethodAsync(
+                code: @"x = (_, _) => 1;",
+                expected: Classifications(Identifier("x"), Operators.Equals, Punctuation.OpenParen, Parameter("_"), Punctuation.Comma, Parameter("_"), Punctuation.CloseParen,
```

Comment:
still treated as parameter, and not as dicard in classification?

Reply:
Even the discard in `_ = 1;` (test above) doesn't have a special classification. I don't think we have a discard classification yet.